doc/DeveloperGuide.md

```diff
@@ -88,6 +88,27 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+#### Use case: Rename tag
+
+**MSS**
+
+1. User requests to rename tag
+2. AddressBook shows a list of persons with the tag
+3. AddressBook requests for confirmation
+4. User confirm to rename
+5. AddressBook renames the tag for all persons with the tag
+Use case ends.
+
+**Extensions**
+
+2a. No person has the tag
+
+> Use case ends
+
+4a. User decided not to rename
```

Comment:
- What happens when the new tag is already present in the address book?
- What happens if the new tag is same as the old tag?